nypl-core-objects.js

```diff
@@ -5,7 +5,7 @@ const FactoryBase = require('./lib/factory_base')
 
 module.exports = (maptype) => {
   switch (maptype) {
-    case 'by-recap-customer-codes':
+    case 'by-recap-customer-code':
```

Comment:
Is this safe? Are there clients using this mapping currently that we should provide backwards support for with a deprecation warning?